go.mod

```diff
@@ -110,7 +110,6 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
-	github.com/opentracing/opentracing-go v1.1.0 // indirect
```

Comment:
I was under the assumption we were using this before ^

Reply:
we are using opencensus, not opentracing I believe.